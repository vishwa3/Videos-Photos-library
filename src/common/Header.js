import React from 'react';
//import { Link, IndexLink } from 'react-router';
import { Link, NavLink } from 'react-router-dom';
const Header = () => (
 /*  <div className="text-center">
    <nav className="navbar navbar-default">
      <NavLink to="/" activeClassName="active">Home</NavLink>
      {" | "}
      <Link to="library">Library</Link>
    </nav>
  </div> */

  <div className="text-center fixNav">
    <nav className="navbar navbar-default ">
      <div className="collapse navbar-collapse">
        <ul className="nav navbar-nav">
          <li className="nav-item active">
            <NavLink to="/" activeClassName="active">Home</NavLink>
          </li>
          <li className="nav-item">
          <Link to="searchImages">Search Images</Link>
          </li>
          <li className="nav-item">
          <Link to="searchVideos">Search Videos</Link>
          </li>
          <li className="nav-item">
            <Link to="library">About</Link>
          </li>
        </ul>
      </div>
    </nav>
  </div> 

);

export default Header;
