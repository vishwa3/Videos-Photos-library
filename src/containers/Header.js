import React from 'react';
import { Link, NavLink } from 'react-router-dom';

const Header = () => (
  <div className="text-center">
    <nav className="navbar navbar-default space">
      <div className="collapse navbar-collapse">
        <ul className="nav navbar-nav vish">
          <li className="nav-item ">
            <NavLink to="/" activeClassName="active">Home</NavLink>
          </li>
          <li className="nav-item">
            <Link to="searchImages">Check out our HD Images</Link>
          </li>
          <li className="nav-item">
            <Link to="searchVideos">Watch Videos</Link>
          </li>
          <li className="nav-item">
            <Link to="about">About</Link>
          </li>
        </ul>
      </div>
    </nav>
  </div>

);

export default Header;
