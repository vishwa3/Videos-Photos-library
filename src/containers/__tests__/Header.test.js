import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../Header';
import Adapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('Header', () => {
  test('snapshot for Header', ()=> {
    const component = renderer.create(<Router><Header /></Router>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})