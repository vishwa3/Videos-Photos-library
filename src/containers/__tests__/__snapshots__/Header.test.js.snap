// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot for Header 1`] = `
<div
  className="text-center"
>
  <nav
    className="navbar navbar-default space"
  >
    <div
      className="collapse navbar-collapse"
    >
      <ul
        className="nav navbar-nav vish"
      >
        <li
          className="nav-item "
        >
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            href="/searchImages"
            onClick={[Function]}
          >
            Check out our HD Images
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            href="/searchVideos"
            onClick={[Function]}
          >
            Watch Videos
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
